go/libkb/identify2_cache.go

```diff
@@ -89,9 +94,21 @@ func (c *Identify2Cache) Delete(uid keybase1.UID) error {
 // Shutdown stops any goroutines in the cache.
 func (c *Identify2Cache) Shutdown() {
 	c.cache.Shutdown()
+	close(c.shutdown)
 }
 
 // DidFullUserLoad is a noop unless we're testing...
 func (c *Identify2Cache) DidFullUserLoad(_ keybase1.UID) {}
 
 func (c *Identify2Cache) UseDiskCache() bool { return true }
+
+func (c *Identify2Cache) periodicLog() {
+	for {
+		select {
+		case <-c.shutdown:
+			return
+		case <-time.After(time.Minute):
+			c.G().Log.Debug("~~~ Identify2Cache num items in memory cache: %d", c.cache.Count())
```

Comment:
what is `~~~`?

Reply:
Rather than setting up a million timers, what about dumping all of this stuff on a `SIGUSR1`?  I guess it doesn't work on windows... But  I'm worried about log spam, and having a bunch of extra goroutines. Maybe not the end of the world.

Reply:
Maybe all of this stuff should be VDL0?

Reply:
Another thought is to just output cache stats after every ~1000 puts.  This way caches that don't grow won't make much noise.

Reply:
I can just remove the logging.

I've been using it in production and these things are very small.

Reply:
`~~~` was for grep.

Reply:
Yeah, so let's kill these if that's OK

Reply:
ok, killed